users.html

```diff
@@ -5,7 +5,6 @@
     <title>Ofek Twitter</title>
     <link rel="stylesheet" href="css/normalize.css" />
     <link rel="stylesheet" href="css/header.css" />
-    <link rel="stylesheet" href="css/footer.css" />
     <link rel="stylesheet" href="css/Users.css" />
 </head>
 <body>
```

Comment:
both here and in "index.html" you used `id="main_wrapper"` - use class if repeating, you can't use same id for more than one element